src/main/scala/net/ripe/rpki/publicationserver/Hashing.scala

```diff
@@ -10,17 +10,28 @@ trait Hashing {
 
   private val base64 = BaseEncoding.base64()
 
-  def stringify(bytes: Array[Byte]) : String = Option(bytes).map {
-    _.map { b => String.format("%02X", new Integer(b & 0xff)) }.mkString
-  }.getOrElse("")
-
-  def stringify(hash: Hash) : String = Option(hash).map { h =>
-    stringify(h.hash.getBytes("UTF-8"))
-  }.getOrElse("")
+  def stringify(bytes: Array[Byte]): String = Option(bytes).map(bytesToHex).getOrElse("")
```

Comment:
Do we need a custom interpretation for efficiency?

There is a thing for sha256 hex strings in Guava (https://guava.dev/releases/21.0/api/docs/com/google/common/hash/Hashing.html#sha256--)

Reply:
It's a custom implementation of binary to hex, not hashing. Actually you remind me to look for the fastest base64 library, as we encode/decide more of it now.

Reply:
Ah I thought you added it to go from binary hash -> hex string. Got it.